crypto/src/hash.rs

```diff
@@ -42,6 +38,8 @@ ffi::ffi_item! {
     }
 }
 
+iroha_ffi::ffi_type! {unsafe impl Transparent for Hash[[u8; Hash::LENGTH]] validated with {|target| Hash::is_lsb_1(target)} }
```

Comment:
it will be FFI serialized as an array. Which means as a pointer in a function call, or by value when part of a struct

Reply:
I think this ought to be a comment in the text.